engine/consensus/matching/engine.go

```diff
@@ -1168,11 +1182,6 @@ func (e *Engine) requestPendingReceipts() (int, uint64, error) {
 		// check if we have an result for the block at this height
 		blockID := header.ID()
 
-		_, ok := knownResultForBlock[blockID]
-		if ok {
-			continue
-		}
-
```

Comment:
this was the culprit why sealing didn't resumed yesterday on mainnet: we had one receipt, so didn't request another one ... but need to receipts to seal 🤦